models/iaf_psc_alpha.h

```diff
@@ -136,21 +142,31 @@ Parameters
 
 The following parameters can be set in the status dictionary.
 
-=============== ======== =============================== ========================================================================
-**Parameter**   **Unit** **Math equivalent**             **Description**
-=============== ======== =============================== ========================================================================
- ``V_m``         mV       :math:`V_{\text{m}}`           Membrane potential
- ``E_L``         mV       :math:`E_\text{L}`             Resting membrane potential
- ``C_m``         pF       :math:`C_{\text{m}}`           Capacity of the membrane
- ``tau_m``       ms       :math:`\tau_{\text{m}}`        Membrane time constant
- ``t_ref``       ms       :math:`t_{\text{ref}}`         Duration of refractory period
- ``V_th``        mV       :math:`V_{\text{th}}`          Spike threshold
- ``V_reset``     mV       :math:`V_{\text{reset}}`       Reset potential of the membrane
- ``tau_syn_ex``  ms       :math:`\tau_{\text{syn, ex}}`  Rise time of the excitatory synaptic alpha function
- ``tau_syn_in``  ms       :math:`\tau_{\text{syn, in}}`  Rise time of the inhibitory synaptic alpha function
- ``I_e``         pA       :math:`I_\text{e}`             Constant input current
- ``V_min``       mV       :math:`V_{\text{min}}`         Absolute lower value for the membrane potenial (default :math:`-\infty`)
-=============== ======== =============================== ========================================================================
+=============== ================== =============================== ========================================================================
+**Parameter**   **Default** **Math equivalent**                    **Description**
```

Comment:
```suggestion
=============== ================== =============================== ========================================================================
**Parameter**   **Default**        **Math equivalent**             **Description**
```

The changes look good, this clearly improves the documentation.
When I build the docs I see the following:
![nest_doc](https://github.com/nest/nest-simulator/assets/44397333/bd8d5558-cbde-47fd-88ea-58ad7059831a)

I think that in the upper box the ```**math``` is incorrect, isn't it (especially in comparison with the lower one)?
I tried to fix this with my suggestion, but doing this with the GitHub interface was a bit tricky, hope it worked.